docs/UserGuide.md

```diff
@@ -284,11 +317,19 @@ _Details coming soon ..._
 
 ## FAQ
 
-**Q**: *placeholder* <br/>
-**A**: *placeholder*
+**Q**: Can I use this application with other people? <br/>
+**A**: As of now, WIFE does not support concurrent users. If you would like to share your WIFE food list with another
+user, install WIFE on their computer and overwrite their data file with the data file created by WIFE in your computer.
```

Comment:
I think this is easily addressed by tags. You can use tags to identify user's items. To show all user item, we can use `listbytag n/wesley` to show all items with wesley tag